client/components/app-promo/index.jsx

```diff
@@ -74,20 +80,31 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		return (
 			<div className="app-promo">
 				<span tabIndex="0" className="app-promo__dismiss" onClick={ this.dismiss } >
 					<Gridicon icon="cross" size={ 24 } />
-					<span className="screen-reader-text">{ this.translate( 'Dismiss' ) }</span>
+					<span className="app-promo__screen-reader-text screen-reader-text">
```

Comment:
I'm all for placeholders, but that would mean doing a complete refactor of all `.screen-reader-text` elements and their styles!
For now I'm going with a simpler `@extend .screen-reader-text;`.